modules/server/variables.tf

```diff
@@ -719,6 +686,34 @@ variable "services" {
   default     = []
 }
 
+#------------#
+# Forseti db #
+#------------#
+
+variable "cloudsql_db_name" {
```

Comment:
Instead of breaking out each item, how about we pass through the entire module? (In Terraform v0.12, entire modules can be used as inputs/outputs.)

Reply:
Can you please share or point me to an example?  Not quite sure I follow.

Reply:
Basically you can do

```
module "server" {
  cloudsql = module.cloudsql
}
```

Reply:
I think I found the right documentation.